pyproject.toml

```diff
@@ -75,7 +78,8 @@ strict = true
 plugins = ["sqlalchemy.ext.mypy.plugin"]
 
 [tool.pytest.ini_options]
-addopts = "-rsx --tb=short"
+pythonpath = "src/sqlacodegen"
+addopts = "-rsx --tb=short --import-mode=importlib"
```

Comment:
Why `--import-mode=importlib`?

Reply:
Removed in #275

Reply:
pytest recommends to use importlib: https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html